pkg/trace/obfuscate/obfuscate.go

```diff
@@ -47,7 +49,10 @@ func NewObfuscator(cfg *config.ObfuscationConfig) *Obfuscator {
 	if cfg == nil {
 		cfg = new(config.ObfuscationConfig)
 	}
-	o := Obfuscator{opts: cfg}
+	o := Obfuscator{
+		opts:                cfg,
+		sqlNewNormalization: config.HasFeature("new_sql_normalization"),
```

Comment:
`new_sql_normalization` is a very ambiguous name. We should use something that indicates what this actually is, maybe `sql_reduce` (because we are further reducing similar queries), or `sql_extra_quantize`.